### DISCUSSION TITLE
Welcome to php-sn-generator Discussions!

### DISCUSSION THREAD

**Discussion by irfaardy** [irfaardy/php-sn-generator#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by devkuros** [comment]
hi.. i want to use 2 different serial number setting.. how to make different config?

**Comment by irfaardy** [comment]
@devkuros use this
example:
```
$sn = new SerialNumber();

$serial1 = $sn->setConfig([	
                        'length' => 5,
                        'segment' => 4,
                        'seperator' => '-',
                        'charset' => "123456789ABCDEFGH"
                    ])
		->generate();


$serial2 = $sn->setConfig([	
                        'length' => 8,
                        'segment' => 4,
                        'seperator' => '-',
                        'charset' => "ABCDEFGHIJKL123"
                    ])
		->generate();

echo $serial1;
echo $serial2;
`````

**Comment by david77185** [comment]
Hello irfaardy

I put the folder into my website

I created a index.php  with html and your script

i can see the main page, but when i click on execute, it happens nothing ; it means i return to an index.php page empty

for me it comes from this is :

form method="POST" action="/**index.php**">

I located your folder into the cloud ( hostinger.com)

Could you help me please ?

**Reply by irfaardy** [reply]
can i see your code?

Sorry I can't reply, because yesterday I had a fever

**Reply by irfaardy** [reply]
@david77185 
no covid, common fever :D

Have you added this infirst line?

```
require "vendor/autoload.php";
use Irfa\SerialNumber\Func\SerialNumber;


```
and 
have you run this first, in root project directory?
`composer require irfa/php-sn-generator`

**Comment by david77185** [comment]
Hello Irfa

Sorry, take care !!

covid ?

In attachment, my index.php but it was your code

I only put your code ( index.php) at the root of my main folder

and past your folder

[cid:ef6362c2-536b-44a0-a918-4a3e14812e0c]
________________________________
De : ɪʀꜰᴀ ᴀʀᴅɪᴀɴꜱʏᴀʜ ***@***.***>
Envoyé : mercredi 9 novembre 2022 03h24
À : irfaardy/php-sn-generator ***@***.***>
Cc : david77185 ***@***.***>; Comment ***@***.***>
Objet : Re: [irfaardy/php-sn-generator] Welcome to php-sn-generator Discussions! (Discussion #1)


can i see your code?

Sorry I can't reply, because yesterday I had a fever

**Reply by irfaardy** [reply]
@david77185 
no covid, common fever :D

Have you added this infirst line?

```
require "vendor/autoload.php";
use Irfa\SerialNumber\Func\SerialNumber;


```
and 
have you run this first, in root project directory?
`composer require irfa/php-sn-generator`

**Comment by david77185** [comment]
no ☺️

I'm going to add it into the php <..>
________________________________
De : ɪʀꜰᴀ ᴀʀᴅɪᴀɴꜱʏᴀʜ ***@***.***>
Envoyé : mercredi 9 novembre 2022 12h33
À : irfaardy/php-sn-generator ***@***.***>
Cc : david77185 ***@***.***>; Comment ***@***.***>
Objet : Re: [irfaardy/php-sn-generator] Welcome to php-sn-generator Discussions! (Discussion #1)


no covid, common fever

Have you added this infirst line?

require "vendor/autoload.php";
use Irfa\SerialNumber\Func\SerialNumber;

**Comment by david77185** [comment]
Into your download.zip, there is no autoload.php ??

________________________________
De : ɪʀꜰᴀ ᴀʀᴅɪᴀɴꜱʏᴀʜ ***@***.***>
Envoyé : mercredi 9 novembre 2022 12h33
À : irfaardy/php-sn-generator ***@***.***>
Cc : david77185 ***@***.***>; Comment ***@***.***>
Objet : Re: [irfaardy/php-sn-generator] Welcome to php-sn-generator Discussions! (Discussion #1)


no covid, common fever

Have you added this infirst line?

require "vendor/autoload.php";
use Irfa\SerialNumber\Func\SerialNumber;

**Reply by irfaardy** [reply]
you have installed composer before?
https://getcomposer.org/download/

then run `composer require irfa/php-sn-generator`
in your project directory

**Comment by david77185** [comment]
Hello irfa

No, because i put yours files on cloud, hostinger.com

Do you think it was posible to use it even i have an internet proveedor?

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
